user/pages/02.development/03.creating-a-skill/docs.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Development Setup
+taxonomy:
+    category:
+        - docs
+---
+
+# Setting up the folders
+
+Once you've come up with your idea for a skill, make a new folder in `mycroft-core/mycroft/skills` with your desired skill name.
+
+For skills, there should generally be three things inside this folder - a folder called `dialog`, a file called `__init__.py`, and a folder called `vocab`. We'll go through these one by one so you know why each is there.
+

```

Comment:
Ensure we have the `test` folder among the others you mentioned. In addition, I'd be really helpful to have a visual file structure showing the folder organization like:

**Folder structure inside `mycroft-core/mycroft/skills`**
```
- my-skill-name
-- dialog
-- test
-- vocab
-- __init__.py
```